docs/en/query_language/agg_functions/parametric_functions.md

```diff
@@ -276,17 +276,137 @@ ORDER BY level
 
 Simply, the level value could only be 0, 1, 2, 3, it means the maxium event action stage that one user could reach.
 
-## retention(cond1, cond2, ...)
+## retention {#retention}
 
-Retention refers to the ability of a company or product to retain its customers over some specified periods.
+Analytical function that shows how much 
+certain conditions, for example, retention of dynamics/level [site traffic statistics](https://yandex.ru/support/partner2/statistics/metrika-visitors-statistics.html?lang=en).
```

Comment:
let's move example with Yandex.Metrica somewhere lower, after we explain what it does in general